src/api/veilarbperson.ts

```diff
@@ -9,7 +10,13 @@ export enum MalformType {
 export interface MalformData {
 	malform: MalformType | null;
 }
-
+// V2 er data fra PDL (målform kommer fra DKIF)
+export interface MalformDataV2 {
+	malformV2: MalformType | null;
+}
 export function fetchMalform(fnr: string): AxiosPromise<MalformData> {
 	return axiosInstance.get(`/veilarbperson/api/person/${fnr}/malform`);
 }
+export function fetchMalformV2(fnr: string): AxiosPromise<MalformDataV2> {
+	return axiosInstance.get(`/veilarbperson/api/v2/person/${fnr}/malformV2`);
```

Comment:
De gamle endepunktene i veilarbperson bruker fnr i pathen til URLen, men dette er egentlig noe vi ønsker å gå vekk ifra. Nye endepunkter burde heller bruker et query parameter for å sende med fnr.
Urlen kan f.eks se slik ut: `/veilarbperson/api/v2/person/malform?fnr=${fnr}`

Reply:
Sneha har bygd om til å bruke query parameter, implementert her også.